react/index.tsx

```diff
@@ -1,8 +1,25 @@
 import React, { Component, Fragment } from 'react'
 import styles from './index.css'
 
-class ExampleTransactionAuthApp extends Component {
-  constructor(props) {
+interface InjectScriptProps {
+  id: string
+  src: string
+  onLoad: (this: GlobalEventHandlers, ev: Event) => any
+}
+
+type Props = {
+  appPayload: any
+}
+
+type State = {
+  scriptLoaded: boolean
+  loading: boolean
+}
+
+class ExampleTransactionAuthApp extends Component<Props, State> {
+  divContainer: React.RefObject<any>
```

Comment:
Como sempre vamos utilizar essa ref em um elemento div, podemos especificar mais essa tipagem com `HTMLDivElement`, o que acha? Assim, vamos ser forçados a validar a existência da ref antes de utilizar o valor de `current`.